src/components/NotificationItem/NotificationItem.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from "react";
+import {useDispatch} from 'react-redux';
+import { notificationActions } from "../../store/slices/notification/slice";
+import Notification from "../../types/Notification";
+import SEVERITY from "../../types/Severity";
+import { notificationMessages } from "../../utils/notificationMessages";
+import {  Close, Container, Message } from "./styles";
+
+type NotificationItemProps ={
+    notification: Notification
+}
+
+const NotificationItem = ({notification}: NotificationItemProps) => {
+    const dispatch = useDispatch();
+
+    const close = () => {
+        dispatch(notificationActions.DELETE_NOTIFICATION(notification.id))
+    }
+
+    useEffect(() => {
+        if (notification.severity === SEVERITY.Info || notification.severity === SEVERITY.Warning) {
```

Comment:
The Info is the success. :) There are Info, Warning and Error notifications.

Reply:
so then the error does not disappear? :D

Reply:
No, it only disappears, when the user closes it. Error messages only appear, if the database is down, and then the app does not work properly (most features do not work at all). So it is really important that the user notices these notifications, that is why they do not disappear by themselves.

Reply:
unless the user blind, he/she will notice a toast notification :D maybe add 1-2 more seconds, if the user is a slow reader

Reply:
Not if the user is looking at another tab, and just listening to the playlist in the background (which I already do often, while using this amazing app :) ). 
And with the party it might happen, that the database breaks down and all the guests quickly realise that sg. is wrong, but the owner does not even notice, because the playback is still going on as usual from their store. The owner would only realise the error message if they look at the app tab just when the song ends.
These error messages will pop up really seldomly, so it is not a great hassle for the users to close them, but if they do pop up, then we should make sure that the user sees them.
That is why I would prefer them not to disappear automatically, what do you think?